Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -121,22 +123,36 @@ const StatusBar = React.createClass({
     },
 
     setBackgroundColor(color, animated?: boolean) {
-      if (Platform.OS !== 'android') {
+      if (Platform.OS === 'windows') {
+        animated = animated || false;
+        StatusBar._defaultProps.backgroundColor = color;
+        StatusBarManager.setColor(processColor(color));    
+      }
+      else if (Platform.OS !== 'android') {
         console.warn('`setBackgroundColor` is only available on Android');
```

Comment:
Maybe it would be better to just attempt to change this to:

if (Platform.OS === 'ios')

And see if the ReactNative team will accept that change.